src/openrct2-ui/interface/InGameConsole.cpp

```diff
@@ -134,9 +134,10 @@ void InGameConsole::ScrollToEnd()
         _consoleScrollPos = std::max<int32_t>(0, static_cast<int32_t>(_consoleLines.size()) - maxLines);
 }
 
-void InGameConsole::RefreshCaret()
+void InGameConsole::RefreshCaret(size_t position = 0)
```

Comment:
The default parameter must be used only on the declaration, this will fail to compile. Consider building it locally before pushing your changes.